src/lib/mojaloop-requests/mojaloopRequests.js

```diff
@@ -72,6 +72,23 @@ class MojaloopRequests {
         this.wso2Auth = config.wso2Auth;
     }
 
+    /**
+     * Executes a GET /authorizations request for the specified transaction request id
+     *
+     * @returns {object} - JSON response body if one was received
+     */
+    async getAuthorizations(transactionRequestId) {
+        const url = `authorizations/${transactionRequestId}`;
+        return this._get(url, 'authorizations');
+    }
+
+    /**
+     * Executes a PUT /authorizations request for the specified transaction request id
+     */
+    async putAuthorizations(transactionRequestId, body, destFspId) {
+        const url = `authorizations/${transactionRequestId}`;
+        return this._put(url, 'parties', body, destFspId);
```

Comment:
@ksatya77 is 'parties' resource type correct here?

Reply:
thx for catching that @bushjames ...updated it now...